purchases/src/main/kotlin/com/revenuecat/purchases/amazon/handler/PurchaseHandler.kt

```diff
@@ -29,7 +30,6 @@ internal class PurchaseHandler(
 ) : PurchaseResponseListener {
 
     private val productTypes = mutableMapOf<String, ProductType>()
-    private val presentedOfferingsByProductIdentifier = mutableMapOf<String, String?>()
```

Comment:
I removed this because it seems it wasn't used. Seems we were passing the offering id in the `AmazonBilling` class instead.